header-rewrite-filter/header_processor.h

```diff
@@ -25,7 +25,7 @@ class DynamicFunctionProcessor : public Processor {
 public:
   DynamicFunctionProcessor() {}
   virtual ~DynamicFunctionProcessor() {}
-  virtual absl::Status parseOperation(absl::string_view function_expression);
+  virtual absl::Status parseOperation(absl::string_view function_expression, bool isRequest);
```

Comment:
Nit: these parameters should probably be `const`. I don't think there's a legitimate reason for the function to ever modify them

Reply:
Can you make this be a const member instead if this will never be modified?